index.js

```diff
@@ -0,0 +1,123 @@
+"use strict";
+
+class WebError extends Error {
+  constructor(status, message) {
+    super(`${status}: ${message}`);
+    this.status = status;
+    return this;
+  }
+}
+
+function minuscule(app) {
+
+  const prodLike = process.env.ENV === 'production';
+
+  const self = {
+
+    use(fn) {
```

Comment:
maybe we should allow the express middleware signature here instead, and if so we don't need the `use` method anymore and it can be swapped with express.

please check below regarding the default error handler

Reply:
I would really like to avoid that, the value proposition of minuscule middleware is that you don't have to think about `next` for normal cases (errors and success).